### PR TITLE
Add K6_PROFILING_ENABLED env in exist option --profiling-enabled

### PR DESCRIPTION
**Pull request by Bablzz**
## What?

Add information K6_PROFILING_ENABLED in exist option `--profiling-enabled`
Close documentation-needed in https://github.com/grafana/k6/pull/3692

## Checklist

- [x] I have used a meaningful title for the PR.
- [x] I have described the changes I've made in the "What?" section above.
- [x] I have performed a self-review of my changes.
- [x] I have run the `npm start` command locally and verified that the changes look good.

- [x] I have made my changes in the `docs/sources/next` folder of the documentation.
- [x] I have reflected my changes in the `docs/sources/v{most_recent_release}` folder of the documentation.
- [ ] I have reflected my changes in the relevant folders of the two previous k6 versions of the documentation (if still applicable to previous versions).

- [x] I have made my changes in the `docs/sources/next` folder of the documentation.

### CODE REVIEW

**Review by mstoykov — APPROVED** [review]

File: docs/sources/v0.50.x/using-k6/k6-options/reference.md [hunk 1]
```diff
@@ -910,9 +910,9 @@ export const options = {
 
 Enables [pprof](https://pkg.go.dev/net/http/pprof) profiling endpoints under the k6's REST API [address](#address). These endpoints help debug and profile k6 itself. k6's REST API should be enabled as well.
 
-| Env | CLI                   | Code / Config file | Default                              |
-| --- | --------------------- | ------------------ | ------------------------------------ |
-| N/A | `--profiling-enabled` | N/A                | `http://localhost:6565/debug/pprof/` |
+| Env                    | CLI                   | Code / Config file | Default                              |
+| ---------------------- | --------------------- | ------------------ | ------------------------------------ |
+| `K6_PROFILING_ENABLED` | `--profiling-enabled` | N/A                | `http://localhost:6565/debug/pprof/` |
```
Comment (mstoykov) [inline docs/sources/v0.50.x/using-k6/k6-options/reference.md:915]:
```suggestion
| Env | CLI | Code / Config file | Default |
| --- | --------------------- | ------------------ | ------------------------------------ |
| N/A | `--profiling-enabled` | N/A | `http://localhost:6565/debug/pprof/` |
```

This actually shoudn't change as v0.50.0 would not support it as it is alredy released

**Review by mstoykov — CHANGES_REQUESTED** [review]

File: docs/sources/v0.50.x/using-k6/k6-options/reference.md (hunk 1 above)
Comment (Bablzz) [reply docs/sources/v0.50.x/using-k6/k6-options/reference.md:915]: oh, yeah, agree. did it according readme.md. thanks!

File: docs/sources/v0.50.x/using-k6/k6-options/reference.md [hunk 2]
```diff
@@ -910,9 +910,9 @@ export const options = {
 
 Enables [pprof](https://pkg.go.dev/net/http/pprof) profiling endpoints under the k6's REST API [address](#address). These endpoints help debug and profile k6 itself. k6's REST API should be enabled as well.
 
-| Env | CLI                   | Code / Config file | Default                              |
+| Env | CLI | Code / Config file | Default |
```
Comment (mstoykov) [inline docs/sources/v0.50.x/using-k6/k6-options/reference.md:913]:
```suggestion
| Env | CLI                   | Code / Config file | Default                              |
```

File: docs/sources/v0.50.x/using-k6/k6-options/reference.md [hunk 3]
```diff
@@ -912,7 +912,7 @@ Enables [pprof](https://pkg.go.dev/net/http/pprof) profiling endpoints under the
 
 | Env | CLI                   | Code / Config file | Default                              |
 | --- | --------------------- | ------------------ | ------------------------------------ |
-| N/A | `--profiling-enabled` | N/A                | `http://localhost:6565/debug/pprof/` |
+| N/A | `--profiling-enabled` | N/A | `http://localhost:6565/debug/pprof/` |
```
Comment (mstoykov) [inline docs/sources/v0.50.x/using-k6/k6-options/reference.md:915]:
```suggestion
| N/A | `--profiling-enabled` | N/A                | `http://localhost:6565/debug/pprof/` |
```

**Review by mstoykov — APPROVED** [review]

**Pull request merged**